### PR TITLE
Fix issues on Windows.

### LINKED ISSUE #127: Disable emoji output on Windows
## Steps to reproduce

1. Run `cargo func new-app`.

## Expected  behavior

The text output does not contain Unicode replacement characters.

## Actual behavior

The text output does contain Unicode replacement characters as emoji isn't yet supported in Windows' command prompt.

## Environment information (OS, versions, etc.)

Windows 10 x64

### PR DESCRIPTION
**Pull request by peterhuene**
The following issues are addressed:

* Fix the printing of emoji that does not look good in a Windows Command
  prompt.
* Remove dependency on OpenSSL entirely; the RPC server doesn't use the
  secure feature anyway (connection is always localhost without TLS).
* Fix the `init` command to copy the rust worker with an `.exe`
  extension.
* Update the README with information relevant to using on Windows.

Fixes #127.

### CODE REVIEW

**Comment by peterhuene** [comment]
Blocked on #145 and will be rebased when that PR is merged.

**Comment by peterhuene** [comment]
With these minimal fixes, it works on Windows 🎉.

Note that for some reason, though, `grpcio-sys` takes quite a long time to build on Windows; much longer than from macOS or Linux.  If it's a pain point for our users, we should probably raise an issue with `grpc-rs` and see if there's something to be done there.

File: README.md [hunk 1]
```diff
@@ -44,17 +43,27 @@ Documentation for the [latest published version](https://docs.rs/azure-functions
 
 # Getting Started
 
+## Install CMake
+
+Install [CMake](https://cmake.org/download/) and ensure the install location is added to the `PATH` environment variable.
```
Comment (rylev) [inline README.md]: `cmake` can easily be installed on macOS and Linux using `brew`/the default package manager. Should we mention that here?

Comment (peterhuene) [reply README.md]: That's a good idea.  I'll update.

File: azure-functions-sdk/src/util.rs [hunk 2]
```diff
@@ -5,15 +5,19 @@ use std::io::Write;
 use std::path::Path;
 
 pub fn print_running(message: &str) {
-    print!("{} {}", "️🚀".cyan(), message);
+    print!(
+        "{} {}",
+        if cfg!(windows) { "->" } else { "️🚀" }.cyan(),
```
Comment (rylev) [inline azure-functions-sdk/src/util.rs]: 🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

File: README.md (hunk 1 above)
Comment (peterhuene) [reply README.md]: Pushed.

**Comment by peterhuene** [comment]
[clabot:check]

**Pull request merged**